pom.xml

```diff
@@ -223,5 +223,32 @@ limitations under the License.
         </pluginManagement>
       </build>
     </profile>
+    <profile>
+      <id>release</id>
```

Comment:
If is called `plexus-release`, see https://github.com/codehaus-plexus/plexus-pom/blob/master/pom.xml#L670

Reply:
Done. Good catch, thank you!